modules/oxxionRtdProvider.js

```diff
@@ -1,34 +1,71 @@
 import { submodule } from '../src/hook.js'
-import { deepAccess, logInfo } from '../src/utils.js'
+import { deepAccess, logInfo, logError } from '../src/utils.js'
+import MD5 from 'crypto-js/md5.js';
```

Comment:
why not use something supported already?

Reply:
We just don't love the include, isnt sha supported in the browser? I forget. md5 is a rather large build side include I believe.

Reply:
https://github.com/mbaersch/cyrb53-hasher is in python; fairly simple to implement in go just asking chatgpt to translate our js function to go

Reply:
Here's an alternative we use at raptive  https://caniuse.com/?search=digest ; it has wide support for sha-1 and sha-256

Reply:
just like I said, I removed it for now as it's not needed yet.